.github/workflows/triage.yml

```diff
@@ -60,6 +61,9 @@ jobs:
   triage:
     runs-on: ubuntu-22.04
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
I'm kinda wary about slowing down triage jobs with an additional clone.

Reply:
Me too. I thought about moving it to the tests.yml file ... but it defeats the "triage" part